CHANGELOG.md

```diff
@@ -2,19 +2,30 @@
 
 ## Next release
 
+### Breaking changes
+
+This PR contains the [PDV refactor (#4111)](https://github.com/hasura/graphql-engine/pull/4111), a significant rewrite of the internals of the server, which did include some breaking changes:
+
+- The semantics of `where` have changed, according to the discussion in [issue 704](https://github.com/hasura/graphql-engine/issues/704#issuecomment-635571407). Namely: an explicit `null` value in a `where` expression will be treated as an error rather than resulting in the expression being evaluated to `True`. Meaning that `delete_users(where: {id: {_eq: $userId}}) { name }` will yield an error if `$userId` is `null` instead of deleting all users.
```

Comment:
IMO it would be nicer to start off with something more precise like, "The semantics of handling `null` value in `where` filter have changed, according to ..."

Note that since we guarantee semver releases, non-trivial breaking changes should bump the major version. But I feel this is a) not that big, b) expected behaviour, we can make do with minor version and annoucement(s).